external/sinon/sinon.js

```diff
@@ -5738,7 +5738,7 @@ if (typeof sinon === "undefined") {
  * Add-on for sinon.fakeServer that automatically handles a fake timer along with
  * the FakeXMLHttpRequest. The direct inspiration for this add-on is jQuery
  * 1.3.x, which does not use xhr object's onreadystatehandler at all - instead,
- * it polls the object for completion with setInterval. Dispite the direct
+ * it polls the object for completion with setInterval. Despite the direct
```

Comment:
anything in `/external` is an external dependency so you'd need to report it in that project.